src/styles/_catalog.scss

```
@@ -9,14 +12,23 @@
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
-    height: 300px;
+    width: $card_width ;
+    height: $card_height;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
+    transition-timing-function: linear;
```

```suggestion
    transition-timing-function: linear;
    transition-property: transform, color;
```
Завжди потрібно вказувати властивість, до якої застосовується transition